### PR TITLE
LPD-39335 Make sure that the object definition will be imported even if its default locale is not available

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://liferay.atlassian.net/browse/LPD-39335

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/definitions/portlet/action/test/ObjectDefinitionExportImportTest.java [hunk 1]
```diff
@@ -42,6 +49,59 @@ public class ObjectDefinitionExportImportTest extends BaseExportImportTestCase {
...
+		testExportImport(
+			"test-object-definition.portuguese-default-locale.json",
+			"test-object-definition.site-default-locale.json",
+			"TESTOBJECTDEFINITIONPORTUGUESE", "TestObjectDefinitionptBR");
+
+		objectDefinitionResource.deleteObjectDefinition(
+			getId("TestObjectDefinitionptBR"));
+
+		// Localized object definition with portuguese removed from the company
+		// available locales
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/definitions/portlet/action/test/ObjectDefinitionExportImportTest.java:78]: // Localized object definition with **the** portuguese **locale** removed from the company
// available locales

Comment (carolmariaabb) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/definitions/portlet/action/test/ObjectDefinitionExportImportTest.java]: Done!

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:report:43642494

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:report:43936051

**Comment by carolmariaabb** [comment]
ci:test:stable

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**